x-pack/plugin/core/template-resources/src/main/resources/logs-otel@mappings.json

```diff
@@ -0,0 +1,97 @@
+{
+  "template": {
+    "mappings": {
+      "date_detection": false,
+      "dynamic": "strict",
+      "properties": {
+        "data_stream.type": {
+          "type": "constant_keyword",
+          "value": "logs"
+        },
+        "data_stream.dataset": {
+          "type": "constant_keyword"
+        },
+        "data_stream.namespace": {
+          "type": "constant_keyword"
+        },
+        "@timestamp": {
+          "type": "date_nanos"
+        },
+        "observed_timestamp": {
+          "type": "date_nanos"
+        },
+        "severity_number": {
+          "type": "byte"
+        },
+        "severity_text": {
+          "type": "keyword"
+        },
+        "body.text": {
+          "type": "match_only_text"
+        },
+        "body.structured": {
+          "type": "flattened"
+        },
+        "attributes": {
+          "type": "object",
+          "dynamic": true,
+          "subobjects": false
+        },
+        "dropped_attributes_count": {
+          "type": "long"
+        },
+        "flags": {
+          "type": "byte"
+        },
+        "trace_id": {
+          "type": "keyword"
+        },
+        "span_id": {
+          "type": "keyword"
+        },
+        "resource": {
```

Comment:
We should probably put the common fields shared across signal types into a component template

Reply:
I'm not certain that we should. I think this should be handled via dynamic mappings in `ecs@mappings` (maybe `semconv@mappings`?) so that the fields are mapped lazily.

Reply:
+1 on keeping the actual attribute fields as dynamic - but here I meant the `resource` and `resource.attributes`, as well as common fields like `@timestamp`, `scope.*`. I believe these should be present on all OTel data streams.

Reply:
gotcha, makes sense